src/pages/storey/containers/item.mdx

```diff
@@ -5,3 +5,87 @@ tags: ["storey", "containers"]
 import { Callout } from "nextra/components";
 
 # Item
+
+An [`Item`] is the simplest container. It stores a single value at a specific key.
+
+An `Item` that's never been `set` is empty, and will return `None` upon a `get`.
+
+<Callout>
+  For a more in-depth look at the `Item` container, check out the [API documentation].
+</Callout>
+
+## Usage examples
+
+### Saving an admin address
+
+```rust template="storage"
+use cw_storey::containers::Item;
+use cw_storey::CwStorage;
+
+let admin: Item<String> = Item::new(0);
+let mut cw_storage = CwStorage(&mut storage);
```

Comment:
Mentioned above already, but it would really be great if we could avoid that for every single storage access